docs/kyma/10-04-common-errors.md

```diff
@@ -69,3 +69,7 @@ To resolve this error, upgrade Tiller. Run:
 ```
 kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/{YOUR_KYMA_VERSION}/installation/resources/tiller.yaml
 ```
+
+## Max number of retries reached during step
```

Comment:
I think this title should remain as it is - this is the error message user gets, so if will be easier to find for him.

Reply:
I have a problem with the fact that the title does not specifically quote the error fully or indicate that this is the error message received and we are left with sth that is not fully grammatically correct
1. we try to avoid contractions (Max)
2. "reached during step"..."during a given step"? "during a step"? (in the error message you have "step xxx")